docs/install_microk8s.md

```diff
@@ -0,0 +1,131 @@
+## 1. Overview
+Kubeflow is a novel open source tool for Machine Learning workflow orchestration on kubernetes. It has great powers, but deploying it may not be so easy, depending on how and where you deploy your kubernetes. 
+This tutorial will show you an easy way to deploy kubeflow using microk8s, a lightweight version of kubernetes, in a few simple steps.
+
+### What you'll learn
+
+* How to deploy microk8s on ubuntu
+* How to deploy kubeflow using microk8s
+* How to access your kubeflow dashboard 
+
+### What you'll need
+* Desktop or Virtual Machine with Ubuntu 16.04 or above
+* A minimum of 4 CPU, 14GB RAM, 50GB Disk (recommended 8 CPU, 32GB RAM, 60GB Disk)
+* Hyper V, if using multipass on a Windows machine (not available on Windows 10 Home)
```

Comment:
@tvansteenburgh: Do we have some way of running this on Windows 10 Home, or are we just SOL?